uber/config.py

```diff
@@ -315,6 +315,12 @@ def _is_intstr(s):
         return s[1:].isdigit()
     return s.isdigit()
 
+if not c.KIOSK_CC_ENABLED:
+    del _config['enums']['door_payment_method']['stripe']
```

Comment:
OK, we do reference them each once in registration.py. If I add a second check for these config options (e.g. `if c.KIOSK_CC_ENABLED and attendee.payment_method == c.STRIPE:`), is that a good solution in your opinion? Or should we do something else? I don't want partial config options hanging around if I can help it, which is why I switched to removing them wholesale.